src/middlewared/middlewared/plugins/kubernetes_linux/update.py

```diff
@@ -369,23 +373,46 @@ async def validate_data(self, data, schema, old_data):
 
         verrors.check()
 
+    @private
+    def wait_on_interface_link_state_up(self, interface):
+        sleep_time, time_waited = 1, 0
+        is_up, error_msg = False, ''
+        while time_waited < IFACE_LINK_STATE_MAX_WAIT:
+            time.sleep(sleep_time)
+            time_waited += 1
```

Comment:
This may be pedantic, but wouldn't we want `time_waited += sleep_time`? In this case since it's just 1 it doesn't really matter.